result/src/commonMain/kotlin/com/github/kittinunf/result/Factory.kt

```diff
@@ -1,17 +1,7 @@
 package com.github.kittinunf.result
 
-inline fun <V> runCatching(block: () -> V?): Result<V?, Throwable> {
-    return try {
-        Result.success(block())
-    } catch (e: Exception) {
-        Result.failure(e)
-    }
-}
+expect inline fun <V> runCatching(block: () -> V?): Result<V?, Throwable>
```

Comment:
Since we can use `doTry` maybe we don't need this to be an expect function any longer?

Reply:
I think that does make sense.